source/manual/rename-a-country.html.md

```diff
@@ -127,24 +127,20 @@ Failing this, there is a [rake task](https://github.com/alphagov/search-api/blob
+  - `international_development_funds`
 
-1. Deploy govuk-content-schemas and Specialist Publisher to production.
+1. Run the rake task [rename_country:all[old_slug,new_slug]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=specialist-publisher&MACHINE_CLASS=backend&RAKE_TASK=rename_country:all[macedonia,north-macedonia]) to republish all the relevant documents.
 
-1. Run the rake task [publishing_api:publish_finder[export_health_certificates]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=specialist-publisher&MACHINE_CLASS=backend&RAKE_TASK=publishing_api:publish_finder[export_health_certificate]) to update the finder options.
+1. Check [the finder](https://www-origin.integration.publishing.service.gov.uk/export-health-certificates?destination_country%5B%5D=north-macedonia) to see the documents appear with the new country filter.
 
-1. Run the rake task [publishing_api:publish_finder[international_development_funds]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=specialist-publisher&MACHINE_CLASS=backend&RAKE_TASK=publishing_api:publish_finder[export_health_certificate]) to update the finder options.
+1. Create and deploy another pull request for GOV.UK Content Schemas ([example](https://github.com/alphagov/govuk-content-schemas/pull/1015)) and Specialist Publisher ([example](https://github.com/alphagov/specialist-publisher/pull/1724)) to remove support for the old country (after the migration).
 
-1. Run the rake task [rename_country:all[old_slug,new_slug]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=specialist-publisher&MACHINE_CLASS=backend&RAKE_TASK=rename_country:all[macedonia,north-macedonia])
+1. Re-publish the finders again, as above.
 
-1. Check [the finder](https://www-origin.integration.publishing.service.gov.uk/export-health-certificates?destination_country%5B%5D=north-macedonia) to see the documents appear with the new country filter
-
-1. Create a final content schemas [pull request](https://github.com/alphagov/govuk-content-schemas/pull/1015) to remove the old country from the schema (after the migration).
-
-1. Create a final [pull request](https://github.com/alphagov/specialist-publisher/pull/1724) to remove the old country from the finders.
-
-1. Deploy govuk-content-schemas and Specialist Publisher to production.
-
-1. Re-publish the finders as above.
+1. Run the rake task [data_migration:update_subscriber_list_tag[field,old_slug,new_slug]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=data_migration:update_subscriber_list_tag[location,old_slug,new_slug]]) to update the matching criteria for any subscriber lists in Email Alert API with keys of:
+  - `location`
+  - `destination_country`
```

Comment:
It seems more consistent to your example if the location argument in the rake task was field to match how it is presented in text.

Reply:
Oops, thanks for spotting. Updated.